### PR TITLE
Foreman Bumped to build on el8 p5

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

**Comment by ehelms** [comment]
We need to omit passenger for right now.

**Comment by zjhuntin** [comment]
https://github.com/theforeman/foreman-packaging/pull/5011

File: packages/foreman/rubygem-powerbar/rubygem-powerbar.spec [hunk 1]
```diff
@@ -84,6 +84,9 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/powerbar.gemspec
 
 %changelog
+* Wed Apr 08 2020 Zach Huntington-Meath <zhunting@redhat.com> - 2.0.1-3
+- Bump to release for EL8
```
Comment (ehelms) [inline packages/foreman/rubygem-powerbar/rubygem-powerbar.spec:88]: Already built for EL8

Comment (ehelms) [inline packages/foreman/rubygem-powerbar/rubygem-powerbar.spec:88]: Already built for EL8, check changelog

**Pull request merged**